File: Appsegundatela/app/src/main/java/com/example/appsegundatela/MainActivity.java
package com.example.appsegundatela;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {


    // abaixo eu criei as seguintes variaveis:
    Button botaosair,botaoabrir;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);

        botaosair = findViewById(R.id.button_sair);
        botaoabrir = findViewById(R.id.button_abrir);

        botaosair.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View view){
                finish();
            }

        });

        botaoabrir.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View view){
                // inicio com intent dps posso colocar qualquer nome
                // nova intenção (obtenha o contexto dessa aplicação(), o nome da tela.class
                Intent Outratela=new Intent(getApplicationContext(),Outratela.class);
                startActivity(Outratela); //inicia a nova atividade
            }
        });

        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;


        });
    }
}

File: Mudandodetelakotlin/app/src/main/java/com/example/mudandodetelakotlin/MainActivity.kt
package com.example.mudandodetelakotlin

import android.content.Intent
import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.example.mudandodetelakotlin.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    //variaveis
    public var loginuser = 0
    public var loginpass = 0


    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {

        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        enableEdgeToEdge()
        setContentView(binding.root)

       loginuser = binding.editTextUser.text.toString().toString()

        binding.buttonAvancar.setOnClickListener {
            //chamar (esta, tela2:: da classe java), essa se usa quando tem muitas tela
        val i = Intent(this, Tela2::class.java)
            startActivity(i) //esse comando executa i

        }


        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }
    }
}